nuxeo-retention-web/ftest/features/governance.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Retention Governance
+
+  As a Record Cleaner, I delete a document under legal hold in governance mode
+
+  Background:
+    Given user "John" exists in group "RecordManager"
+
+  Scenario: Record Cleaner Role
+    Given I have a File document
+    And "John" has ManageLegalHold permission on the document
+    When I login as "John"
+    And I browse to the document
+    Then I can see the "my document" document
+    When I set a legal hold on the document with description "My legal hold"
+    Then I see the document is under legal hold
+    And I cannot see the trash button
+    And I logout
+    Given I have the following groups
+      | name                    | label          |
+      | NuxeoRecordCleaners     | RecordCleaners |
+    And user "Jack" exists in group "NuxeoRecordCleaners"
+    When I login as "Jack"
+    And I have the following permissions to the documents
+      | permission  | path            |
+      | Everything  | /default-domain |
+    When I browse to the document
+    Then I can trash current document
+    When I can navigate to trash pill
+    # Adding this workaround because the Trash data-table is taking too long to load
```

Comment:
I'm not sure about the workaround, couldn't we wait on something on the table to load (or change the display)? If not, then well, go with the workaround.

Reply:
Yes, definitely 😄